src/firefly/js/ui/DefaultSearchActions.js

```diff
@@ -276,7 +277,17 @@ function searchSimbad(cenWpt,radius) {
     };
     const url = base + '?' + new URLSearchParams(params).toString();
     const request= makeFileRequest('Simbad', url);
+    const {tbl_id}= request.META_INFO;
     dispatchTableSearch(request);
+    onTableLoaded(tbl_id).then( (table) => {
+        if (table.error) {
+            const lon= sprintf('%.5f',cenWpt.getLon());
+            const lat= sprintf('%.5f',cenWpt.getLat());
+            const radStr= sprintf('%.5f',Number(radius));
+            const error= `Simbad failed to find any data for ${lon}, ${lat} and ${radStr} degrees`;
+            dispatchTableUpdate({...table,error});
+        }
```

Comment:
@vandesai1 made a good point about inconsistent messages. If you can distinguish between "no data found" vs an actual error, it's better to return an empty table instead of an error.

To return an empty table, update it with { error: null, totalRows: 0 }.

Reply:
I will take a closer look at the Simbad error message and see it I can tell.